target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -142,6 +150,10 @@ case "$FIRMWARE" in
 	unielec,u4019-32m)
 		caldata_extract "0:ART" 0x5000 0x2f20
 		;;
+	engenius,emr5000)
+		ath10kcal_extract "0:ART" 0x5000 0x2f20
+		ath10kcal_patch_mac_crc $(macaddr_add "$(cat /sys/class/net/eth0/address)" 1)
```

Comment:
Have you ever tested what happens if you drop the ath10kcal_patch_mac_crc lines in this file? Will the correct address be assigned anyway (i.e., is the correct address stored in the calibration data?).